tests/test_tutorials.py

```diff
@@ -0,0 +1,85 @@
+"""
+This test file is part of the openPMD-viewer.
+
+It makes sure that the tutorial notebooks run without error.
+
+Usage:
+This file is meant to be run from the root directory of openPMD-viewer,
+by any of the following commands
+$ python tests/test_tutorials.py
+$ py.test
+$ python setup.py test
+"""
+import os, urllib, tarfile
+import re
+
+def test_tutorials():
+    """Test all the tutorial notebooks"""
+
+    # Go to the relative path where all tutorial notebooks are
+    os.chdir( 'tutorials' )
+    tutorial_notebooks = [ filename for filename in os.listdir('./') \
+                           if filename[-6:] == '.ipynb' ]
+
+    # Loop through the tutorials and test them
+    for notebook_name in tutorial_notebooks:
+
+        # Do a first pass where only the non-IPython features are tested.
+        # (This gives better debugging information.)
+        # The notebook is converted to a standard Python script and
+        # run directly with `execfile`
+        script_name = notebook_name[:-6] + '.py'
+        os.system( 'ipython nbconvert --to=python %s' %notebook_name )
+        clean_ipython_features( script_name )
+        execfile( script_name )
```

Comment:
do you really want to *source* the other script into the current one? Or just execute an other script?
In the latter case, execfile is not the right choice.